phoenix_live_view.exs

```diff
@@ -0,0 +1,81 @@
+Application.put_env(:phoenix, :json_library, Jason)
```

Comment:
Personally, I don't think it's relevant to the example and goes against being minimalist for it.

I could be biased because I hate auto-opening the browser though :grinning:

Reply:
sounds good!